common-custom-user-data-gradle-plugin/src/main/java/com/gradle/Utils.java

```diff
@@ -92,6 +97,14 @@ static Properties readPropertiesFile(String name) {
         }
     }
 
+    static String readFile(String name, ProviderFactory providers, Gradle gradle) {
+        Provider<File> file = providers.provider(() -> new File(name));
```

Comment:
🙈

Reply:
What do you mean by this comment, @runningcode ?